examples/video/index.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: page_v2
+title: Prebid Video Examples
```

Comment:
I see this page in the left nav, but I can't click on it.

Reply:
Those pages really shouldn't be accessible or viewable. All examples should be accessed through the left nav. I made index pages for examples/index.html and examples/video/index.html in case some people had bookmarked them. They now contain links to the examples on the site. I want to discourage people from circumventing the left nav. If I didn't think these pages were being bookmarked I would have deleted them entirely.